README.md

```diff
@@ -56,7 +56,7 @@ npm install @google-cloud/bigtable
 
 ```javascript
 // Imports the Google Cloud client library
-const Bigtable = require('@google-cloud/bigtable');
+const {Bigtable} = require('@google-cloud/bigtable');
 
 const bigtable = Bigtable();
```

Comment:
Should we pop a quick `new` in front of `Bigtable()` while we're in here?

Reply:
A quick scroll down shows a few others as well.

Reply:
YES